test/test_utils.jl

```diff
@@ -294,3 +294,41 @@ module DummyRefShapes
         )
     end
 end
+
+############################################################
+# Inverse parametric mapping ξ = ϕ(x) for testing hessians #             
+############################################################
+function function_value_from_physical_coord(interpolation::Interpolation, cell_coordinates, X::Vec{dim,T}, ue) where {dim,T}
+    n_basefuncs = getnbasefunctions(interpolation)
+    scalar_ip = interpolation isa Ferrite.ScalarInterpolation ? interpolation : interpolation.ip
+    @assert length(ue) == n_basefuncs
+    ξ = MAPPING(scalar_ip, cell_coordinates, X)
+    u = zero(typeof(shape_value(interpolation, ξ, 1))) #Is there a utility function for this init?
```

Comment:
Only for `AbstractValues`, but could be simplified to this I guess?
```suggestion
    u = zero(shape_value(interpolation, ξ, 1))
```